.github/workflows/canary-integration-test.yml

```diff
@@ -59,6 +59,11 @@ jobs:
         kubectl -n rook-ceph cp deploy/examples/create-external-cluster-resources.py $toolbox:/etc/ceph
         timeout 10 sh -c "until kubectl -n rook-ceph exec $toolbox -- python3 /etc/ceph/create-external-cluster-resources.py --rbd-data-pool-name replicapool; do echo 'waiting for script to succeed' && sleep 1; done"
 
+    - name: test external script for erasure coded rbd pool
```

Comment:
The CI only creates a single OSD, but for the EC pool we need 3 OSDs. Therefore, any operation that actually requires the pool to be functional will hang since ceph cannot read or write to the pool. With #9363 I will disable this test for now, and separately we will need to see if we can get 3 OSDs running in the CI so this test can be created again.

Reply:
Okay, but my question is how it is not failing always sometimes both ec and replicapool are created and sometimes not.

Reply:
When the pool controller creates the pool, there are basically two steps:
1. Create the pool
2. Init the pool

The EC pool is always created, but then hangs on initializing the pool. So if the EC pool is the last pool created, everything will appear to work since the initialization isn't critical to the test. But if any other pool hasn't been reconciled yet, it will never get created.